app/controllers/auth/PayeAuthAction.scala

```diff
@@ -38,7 +38,7 @@ class PayeAuthActionImpl @Inject()(override val authConnector: AuthConnector,
     implicit val hc: HeaderCarrier =
       HeaderCarrierConverter.fromHeadersAndSession(request.headers, Some(request.session))
 
-    authorised(ConfidenceLevel.L200 and AuthNino(hasNino = true))
+    authorised(ConfidenceLevel.L200 and AuthNino(hasNino = true) and CredentialStrength(CredentialStrength.strong))
```

Comment:
Need to do a MFA uplift if this fails and tests around it

You will also need a test for both, just so you know Individuals should go through MFA then IV.

Reply:
As discussed, in the case for credential strength other than strong, we'll just be catching the exception and logging that 👍